src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -2409,6 +2287,10 @@ public static void main(String[] args) throws Exception {
     i01.execGesture("BREAKITdaVinci()");
   }
 
+  public void startVinMoov() {

```

Comment:
This is way beyond the scope of servo fixes.  why are we changing this in the servo fix pull request?

Reply:
if you run down the rabbit hole, you'll find servo, servoevents, IK and jme intertwined.
I'm untangling them.